Shared/en.lproj/Localizable.strings

```diff
@@ -125,19 +125,19 @@
 "The app needs to be open to work." = "The app needs to be open to work.";
 
 /* No comment provided by engineer. */
-"This is a public service (disables private key)" = "This is a public service (disables private key)";
+"This is a public service (disables private key)" = "Public service. (Turns off the private key.)";
 
 /* No comment provided by engineer. */
 "Use Bridges" = "Use Bridges";
 
 /* No comment provided by engineer. */
-"You added files from outside while the server is running." = "You added files from outside while the server is running.";
+"You added files from outside while the server is running." = "You added files from outside while running the OnionShare.";
```

Comment:
```suggestion
"You added files from outside while the server is running." = "You added files from elsewhere while running the OnionShare.";
```
"The outside"? Is what exactly?

Reply:
Ha, you got me there! That one's from me personally. 🙈

> elsewhere

Yes! Thank you, definitely better.

> while offering the OnionShare

I agree, "server is running" is all about technicalities, which the user might not understand, but this sound strange. Also, I'd rather not mention the app name here. The base code is white-labeled and other names might sound even more strange.

How about

> You added files from elsewhere while already sharing.

Reply:
Strange and unprecedented. I mis-remembered where I got shut down trying to do that before.

```suggestion
"You added files from outside while the server is running." = "You tried adding files to an already active share.";
```
Introduces "share". I can't find out what the nomenclature is already.

As a lesser option, there isn't any "while in sharing mode" to refer to with that just being "Sharing".

There is "publishing" in the Android app. https://hosted.weblate.org/translate/onionshare/onionshare-android/nb_NO/?checksum=811516b647875dde
Not sure what that is though.

Reply:
Aaah, yes, no. To give you more context:

One *cannot* add files, after the sharing started from *within* the app.
However, we support a so-called "Share Extension", with which users can share stuff from *other* apps *into* OnionShare.

Hence the "outside" or "elsewhere". Now you dropped that completely.

For even more context:

This text shows up in an alert box, when files were added and you return to OnionShare, while it is still sharing: (with current text)

```
+------------------------------------+
|       New files while running    x |
+------------------------------------+
| You added files from outside while |
| the server is running.             |
|                                    |
| Please note, that this will only   |
| take effect, after you restart.    |
+------------------------------------+
|                  OK                |
+------------------------------------+
```

Additionally, we have 2 modes:
- "Sharing", where an `index.html` is generated which links to all the content.
- "Hosting", where *no* `index.html` is generated and you're just running a web server with your content.

-> Would be good, if the text works both ways.

Technically, both is "serving". But that's not something, a normal user is acquainted with. What's good wording?

Reply:
While it is not possible to stop and start with added files, the distinction of where files came from is not a meaningful distinction?

Is it somehow possible to add files if the hosting/sharing was initiated with the share extention?
If not, I think the new suggestion works, and that saves the "please note" part (and also doesn't bait the user into restarting the device.)

Reply:
Sorry, but I don't think I completely understand what you wanted to say here.

Anyhow, just to clarify: You cannot start sharing via the Share Extension. But you can add files to the app, while it is already serving files. 
Restarting would mean that a new address would be generated. So you want to do that very deliberately.

I'll change this to my suggestion. Thanks for your input!